x-pack/plugins/canvas/public/components/workpad_header/fullscreen_control/fullscreen_control.js

```diff
@@ -16,6 +16,10 @@ export class FullscreenControl extends React.PureComponent {
     if (enterFullscreen || exitFullscreen) {
       this.toggleFullscreen();
     }
+
+    if (action === 'PAGE_CYCLE_TOGGLE') {
```

Comment:
Does this not exist as a `const` somewhere?

Reply:
All the keymappings live in `kibana/x-pack/plugins/canvas/public/lib/keymap.js`, and are all just referred to by their name (see the other action handlers a few lines above).